src/main/resources/lib/credentials/select/select.js

```diff
@@ -260,6 +260,21 @@ Behaviour.specify("INPUT.credentials-select", 'credentials-select', -100, functi
   }).bind($(x));
   x.onchange();
 });
+Behaviour.specify("DIV.include-user-credentials", 'include-user-credentials', 0, function (e) {
+    e.querySelector("input[name='includeUser']").addEventListener('click', function (evt) {
+        var caution = e.querySelector('span.user-credentials-caution');
+        if (this.checked) {
+            caution.classList.remove('hidden');
+        } else {
+            caution.classList.add('hidden');
+        }
+    });
+    // simpler version of f:helpLink using inline help text
+    e.querySelector('a.help-btn').addEventListener('click', function (evt) {
+        this.remove();
```

Comment:
Before anyone says "hey wait a minute, isn't this function a little too new to use across all browsers?", let me point out that having the question mark not remove itself after clicking it does no harm.

Reply:
Thank you for the highlight here :)

Actually the `remove` function is not browser dependent, it's provided by prototype. The browser methods are `parentNode` and `removeNode` which are supported everywhere (even on IE6 🎉).
Side note, if there was a problem with that method, it will be harmful because the second line will not be executed and thus, the help block not being displayed.

---

:warning: But there is another problem here. I tested with IE and even with Chrome, the method seems to be called twice and thus, log a javascript error in the console. You cannot remove an element that was already removed.
```
Uncaught TypeError: Cannot read property 'removeChild' of null
    at remove (prototype.js:1992)
    at HTMLAnchorElement._methodized [as remove] (prototype.js:461)
    at HTMLAnchorElement.<anonymous> (select.js:274)
```

This problem is due to the hack at the bottom of the script (👿).

Suggestion: 
```suggestion
        this.parentNode && this.remove();
```

That should avoid the error to be displayed every time you click on the icon.

---

The "new" UI/UX introduced here is not consistent with the rest of the project.
1) The icon normally does not go away after a click
2) The icon should allow to re-click to close
3) Normally the icon is at the right of the screen
4) The icon is glued with the warning message as displayed in the following screenshot
5) There is no link to the plugin providing that info text (very useful when trying to debug a problem)

<details>

<summary>Screenshot</summary>

![Screenshot_2019-08-23_132728_001](https://user-images.githubusercontent.com/2662497/63589610-e5f9a980-c5a9-11e9-9e6a-9422d17f4c64.png)


</details>

As `c:select` is not using the "standard" table layout used by the rest of the component, we cannot use the "standard" help here, thus we have troubles. With this change, we are going even further in the non-standard direction.

Personally I will not block the PR due to the UI inconsistency, but it's sad to have to use such hacky way just to have a help message. cc @josephbrueggen.